pkg/security/ebpf/c/exec.h

```diff
@@ -576,7 +726,7 @@ int __attribute__((always_inline)) parse_args_and_env(struct pt_regs *ctx) {
     // call it here before the memory get replaced
     fill_span_context(&syscall->exec.span_context);
 
-    bpf_tail_call_compat(ctx, &args_envs_progs, syscall->exec.next_tail);
+    //bpf_tail_call_compat(ctx, &args_envs_progs, syscall->exec.next_tail);
```

Comment:
should we delete this? or at least have a comment about when why we're not deleting this just yet?

Reply:
Just deleted it.